lib/usd/utils/MergePrimsOptions.cpp

```diff
@@ -0,0 +1,191 @@
+        TF_CODING_ERROR(
+            "Dictionary is missing required key '%s' or key is "
+            "not a vector of tokens",
+            key.GetText());
+        return def;
+    }
+
+    MergeMissing missingHandling = MergeMissing::None;
+
+    const auto tokens = VtDictionaryGet<std::vector<VtValue>>(options, key);
+    for (const auto& token : tokens) {
+        if (UsdMayaMergeOptionsTokens->None == token)
+            missingHandling = missingHandling | MergeMissing::None;
+        if (UsdMayaMergeOptionsTokens->Create == token)
+            missingHandling = missingHandling | MergeMissing::Create;
+        if (UsdMayaMergeOptionsTokens->Preserve == token)
+            missingHandling = missingHandling | MergeMissing::Preserve;
+        if (UsdMayaMergeOptionsTokens->All == token)
+            missingHandling = missingHandling | MergeMissing::All;
+    }
+
+    return missingHandling;
+}
+
+} // namespace
+
+/* static */
+const VtDictionary& MergePrimsOptions::getDefaultDictionary()
+{
+    static VtDictionary   d;
+    static std::once_flag once;
+    std::call_once(once, []() {
+        const auto _token = VtValue(TfToken());
+        const auto _tokenVector = VtValue(std::vector<VtValue>({ _token }));
+
+        d[UsdMayaMergeOptionsTokens->verbosity]
+            = VtValue(std::vector<VtValue>({ VtValue(UsdMayaMergeOptionsTokens->Default) }));
+        d[UsdMayaMergeOptionsTokens->mergeChildren] = false;
+        d[UsdMayaMergeOptionsTokens->ignoreUpperLayerOpinions] = false;
+        d[UsdMayaMergeOptionsTokens->propertiesHandling]
+            = VtValue(std::vector<VtValue>({ VtValue(UsdMayaMergeOptionsTokens->All) }));
+        d[UsdMayaMergeOptionsTokens->primsHandling]
+            = VtValue(std::vector<VtValue>({ VtValue(UsdMayaMergeOptionsTokens->All) }));
+        d[UsdMayaMergeOptionsTokens->connectionsHandling]
+            = VtValue(std::vector<VtValue>({ VtValue(UsdMayaMergeOptionsTokens->All) }));
+        d[UsdMayaMergeOptionsTokens->relationshipsHandling]
+            = VtValue(std::vector<VtValue>({ VtValue(UsdMayaMergeOptionsTokens->All) }));
+        d[UsdMayaMergeOptionsTokens->variantsHandling]
+            = VtValue(std::vector<VtValue>({ VtValue(UsdMayaMergeOptionsTokens->All) }));
+        d[UsdMayaMergeOptionsTokens->variantSetsHandling]
+            = VtValue(std::vector<VtValue>({ VtValue(UsdMayaMergeOptionsTokens->All) }));
+        d[UsdMayaMergeOptionsTokens->expressionsHandling]
+            = VtValue(std::vector<VtValue>({ VtValue(UsdMayaMergeOptionsTokens->All) }));
+        d[UsdMayaMergeOptionsTokens->mappersHandling]
+            = VtValue(std::vector<VtValue>({ VtValue(UsdMayaMergeOptionsTokens->All) }));
+        d[UsdMayaMergeOptionsTokens->mapperArgsHandling]
+            = VtValue(std::vector<VtValue>({ VtValue(UsdMayaMergeOptionsTokens->All) }));
+    });
+
+    return d;
+}
+
+MergePrimsOptions::MergePrimsOptions(const VtDictionary& options)
```

Comment:
To be able to make the data members const and initialize them from the constructor initializer list, you'll probably have to define a private constructor with a different signature, and delegate the construction to it:

MergePrimsOptions::MergePrimsOptions(const VtDictionary& filledOptions, bool dummy) : verbosity(parseVerbosity(filledOptions, UsdMayaMergeOptionsTokens->verbosity)), mergeChildren(...)